docs/packages/omc3/analysis.md

```diff
@@ -770,7 +770,37 @@ trackone.sdds.bad_bpms_y	trackone.sdds.liny
     To use these, refer to the [Plot Spectrum][plot_spectrum] API documentation.
 
 ??? question "Column Nomenclature"
-    TODO: add a reference with the meaning of each column in the important files (the `.lin*` ones)
+    The `*.amps[xy]`, `*.freqs[xy]` and `*.lin[xy]` files in the harmonic analysis output are **TFS** files.
+
+    The `*.freqs[xy]` files contain for each BPM in column format the frequencies of the resonance lines detected in the spectrum, for respectively the horizontal (`.freqsx`) and vertical (`.freqsy`) planes while the `*.amps[xy]` files contain the amplitudes of said resonance lines.
+    This means in the column of a given BPM, the `nth` value in the `.amps[xy]` file corresponds to the amplitude of the resonance line located at the frequency given by the `nth` value in the column of the same name in the `.freqs[xy]` file.
+    Plotting the spectrum from these files would simply go as:
+    ```python
+    import tfs
+    import matplotlib.pyplot as plt
+
+    ampsx = tfs.read("harpy_output/trackone.sdds.ampsx")
+    freqsx = tfs.read("harpy_output/trackone.sdds.freqsx")
+
+    for bpm in ampsx.columns:
+        if bpm in freqsx.columns:  # safety check but no reason it wouldn't be there
+            plt.plot(freqsx[bpm], ampsx[bpm], ".-") 
+            # plt.stem(freqsx[bpm], ampsx[bpm])  # would be more accurate but might stress your system
+
+    plt.xlim(0, 0.5)
+    plt.xlabel(r"$Q_x$")
+    plt.ylabel("Amplitude")
+    plt.title("Horizontal Spectrum")
+    ```
+
+    The `*.lin[xy]` files contain various data (in columns) computed for each BPM (in rows) as some summary information.
+    Some column names are explicit: `BPM_RES` contains the determined BPM resolution, `CO` the closed orbit value at said BPM and `PK2PK` is the peak-to-peak value of oscillations registered by the given BPM.
+    `TUNE[XY]`, `AMP[XY]` and `MU[XY]` are the detected tune, the amplitude of the tune line and the phase of the tune line for said BPM, respectively.
+    
+    Some columns are named less explicitely but important for later optics calculations.
```

Comment:
"The other columns are describe the phase (`PHASE`), amplitude (`AMP`) and frequency (`FREQ`) of the line identified by the number in the column name. Underscores represent a minus sign here."

Reply:
This is much better, thanks!